### ISSUE TITLE
Add setup guide for Insomnia?

### ISSUE THREAD

**Issue by react-learner** [meilisearch/documentation#713]
We already have a Postman collection and [setup guide](https://docs.meilisearch.com/resources/postman_collection.html) in the docs. Maybe we should have a similar resource for Insomnia, which is a similar product.

What do you think @curquiza @eskombro @bidoubiwa ?

**Comment by fharper** [comment]
I agree, Insomnia have a different goal than Postman, but you can do most of things you do on Postman with Insomnia (last time I checked).

**Comment by curquiza** [comment]
I completely agree with this! Plus, Insomnia is open source!
It's been a while I want to try it! So, I'm volunteering to give you feedback on it and to provide the Insomnia collection if it fits our need 😁
FYI, If I'm not wrong, @MarinPostma already uses it (sorry if I'm wrong and I pinged you for nothin Marin)

**Comment by MarinPostma** [comment]
I used it in the past, but I switched to httpie

**Comment by fharper** [comment]
@MarinPostma can you use it to manage collections? I use httpie "just" as a curl replacement...

**Comment by MarinPostma** [comment]
No need I use bash aliases/variables for the rest, it suits my needs

**Comment by curquiza** [comment]
I've just tested: the collection we provide for Postman can also be imported into Insomnia 🎉 
No need to create a new collection.

**Comment by curquiza** [comment]
The contrary is not true: I cannot import Marin's collection for Insomnia in my Postman app

**Comment by harisarang** [comment]
Hey @dichotommy I would like to take up this issue can you assign this to me?

**Comment by curquiza** [comment]
Shouldn;t we close this one since both collections are working for insomnia and postman?

**Comment by curquiza** [comment]
Discussed with @guimachiavelli 
Indeed, using Postman collection already works, so we decided to close the issue.
Feel free to let us know if it's something you need 😊
